topics/groovy/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: groovy-lang
+created_by: James Strachan
+display_name: Groovy
+logo: groovy.png
```

Comment:
The groovy.png image is currently bigger than the required 288x288 size. Can you resize it to fit?